dev-java/bnd-annotation/bnd-annotation-7.0.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="biz.aQute.bnd:biz.aQute.bnd.annotation:${PV}"
```

Comment:
it seems we already have this package in the tree under `dev-java/aqute-bnd-annotation`. if so, it would be imo better to first rename `dev-java/aqute-bnd-annotation` to `dev-java/bnd-annotation`.